sources/settings-defaults/aws-ecs-1/defaults.d/40-aws-creds.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/aws-creds.toml
```

Comment:
Isn't a migration needed to remove the metadata on a downgrade?

Reply:
> Isn't a migration needed to remove the metadata on a downgrade?

We have to be very careful with migrations now that affect some but not all variants, since we can't do variant-specific migrations any more until we build the scaffolding for that.

I expect that this would be covered by the existing migrations for v1.11.0 but it wouldn't hurt to test a downgrade from a version with this change into 1.26.1.